tests/components/goodwe/test_config_flow.py

```diff
@@ -0,0 +1,102 @@
+"""Test the Goodwe config flow."""
+from unittest.mock import AsyncMock, patch
+
+from goodwe import InverterError
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.goodwe import const
```

Comment:
I don't think importing `const` as a namespace facilitates reading of the code. I think in this case it is better to import the individual constants, or rename the namespace  `const as goodwe_const`.
```suggestion
from homeassistant.components.goodwe.const import CONF_NETWORK_RETRIES, CONF_NETWORK_TIMEOUT, DOMAIN...
```

Reply:
Done